crates/uv/tests/pip_show.rs

```diff
@@ -85,14 +85,7 @@ fn show_requires_multiple() -> Result<()> {
     );
 
     context.assert_command("import requests").success();
-    let filters = [(
-        r"Location:.*site-packages",
-        "Location: [WORKSPACE_DIR]/site-packages",
-    )]
-    .to_vec();
```

Comment:
We should avoid filtering based on other context in the output (i.e. "Location: ") unless absolutely necessary.